code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -146,7 +140,6 @@
 
 /obj/item/storage/firstaid/radiation/Initialize()
```

Comment:
need to delete these Initialize() procs altogether if all they do is call `..()`